models/src/lib.rs

```diff
@@ -1,5 +1,6 @@
 pub mod agent;
 pub mod apiserver;
 pub mod constants;
+pub mod controller;
```

Comment:
The mods inside models have a very similar structure for the crates under brupop, how do you design what should be in models and what should be in other lib+bin?

Reply:
So far, anything that is written to Kubernetes as an object, or object models that are shared between many crates end up in the `models` crate. There's definitely some discretion to placing code.